server/controllers/SolutionController.js

```diff
@@ -2,62 +2,79 @@ const db = require("../models/index");
 const Solution = db.Solution;
 const Comment = db.Comment;
 
+// TODO: extract try-catch error logic into a function / middleware
```

Comment:
@o-bm What's the best way to do this?

Reply:
Not sure maybe ask @arthurhohinng

Reply:
discussed in 1:1 but create a generic function middleware function with parameters passed in

https://medium.com/@arunchaitanya/understanding-normal-middleware-and-error-handling-middleware-in-express-js-d3ecbd9b9849

Can do this in a different PR